plugins/inventory/now.py

```diff
@@ -588,10 +588,13 @@ def parse(self, inventory, loader, path, cache=True):
                 )
 
                 referenced_dict = dict((x["sys_id"], x) for x in referenced_records)
+                processed_records = []
```

Comment:
Could you add a simple comment here on *why* do we need this kind of logic, please? It will help us better understand this code in the future